wiki/Daily_challenge/en.md

```diff
@@ -0,0 +1,65 @@
+# Daily Challenge
+
+The **daily challenge** is a multiplayer mode in osu!(lazer) where players are able to build up a *daily* streak by passing beatmaps consecutively with increasing difficulty.
+
+Each day will introduce one new map from the [featured artist](/wiki/People/Featured_Artists) listing with increased difficulty. Regularly after one week however, the difficulty would be reset again.
+
+Sometimes, the maps could come with a mod forced onto users, which required them to pass the map with the selected mod.
```

Comment:
```suggestion
Sometimes, the maps will come with a mod forced onto the users, which required them to play the map with the selected mod.
```
Would definitely change pass to play since the goal of the daily challenge is not to pass each map but to get a score as high as possible.

Reply:
> Would definitely change pass to play since the goal of the daily challenge is not to pass each map but to get a score as high as possible.

Thats a secondary goal technically - To get your rewards (streaks) you need to **pass** the beatmap with any mods you desire (+ the one the map is forced with), getting a score thats high as possible is a goal not everyone will have, and is a very different for each map, whatever if that is because of the song, overall map, forced mod, difficulty and anything else that affects gameplay expirience for the user.

I can switch it to this suggestion (at the moment of writing im currently with Walavouchey's suggestion), but I will need more input from others to decide on the final version.